modules/tile-converter/src/deps-installer/deps-installer.js

```diff
@@ -0,0 +1,22 @@
+/* global process */
+
+import {load} from '@loaders.gl/core';
+import {ZipLoader} from '@loaders.gl/zip';
+import {writeFile} from '../lib/utils/file-utils';
+import {join} from 'path';
+
+const PGM_LINK =
```

Comment:
Is it safe to depend on this link or should we copy this somewhere? Does the license allow us to copy it somewhere?